django_rq/management/commands/rqpool.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+
+from rq.serializers import resolve_serializer
+from rq.worker_pool import WorkerPool
+from rq.logutils import setup_loghandlers
+
+from django.core.management.base import BaseCommand
+
+from ...jobs import get_job_class
+from ...utils import configure_sentry
+from ...queues import get_queues
+from ...workers import get_worker_class
+
+
+class Command(BaseCommand):
```

Comment:
Mind changing the command to `./manage rqworker-pool`? The original RQ command is `worker-pool` so I'd like the command to be symmetric.